gdal/doc/source/drivers/vector/geojson.rst

```diff
@@ -251,6 +251,7 @@ Layer creation options
    must be written as the 'id' member of Feature objects.
 -  **ID_TYPE**\ =AUTO/String/Integer. (OGR >= 2.3) Type of the 'id'
    memer of Feature objects.
+-  **ID_GENERATE**\ =YES/NO. (OGR >= 3.0.4) Auto-generate feature ids
```

Comment:
```suggestion
-  **ID_GENERATE**\ =YES/NO. (OGR >= 3.1) Auto-generate feature ids
```

Reply:
@rouault

Reply:
what did you mean by that ping ? I suggested 3.1 since this will go through master and that's what master will be tagged in a few weeks

Reply:
Just that I added the unit tests
I am already packaging gdal myself anyway, so I am not in a hurry for a new release

Reply:
sure, but I want you to commit my suggested change since that won't definitely not go to 3.0.4  since it has already been released :-)